ntbs-service/wwwroot/css/header.scss

```diff
@@ -128,14 +128,21 @@ body:not(.js-enabled) {
 }
 
 .header-logo {
-    padding: 0 40px 5px 0;
+    padding: 4px 40px 5px 0;
     margin: 0;
-    width: 200px;
     box-sizing: content-box;
+    width: 240px;
 
     img {
-        height: 30px;
-        width: 200px;
+      width: 100%;
+      height: auto;
+    }
+}
+
+@media (min-width: 400px) {
+    .header-logo {
+      padding-top: 0;
+      width: 305px;

```

Comment:
Maybe this would be better as a percentage of the viewport width, so that it scales well for even smaller screens?

(Honestly, I think the position of the menu dropdown is a bit wrong - rather than being absolutely positioned in the top corner, I think it ought to appear at the top of the menu list, and then we wouldn't have this problem at all. But it's probably not a change that's worth making.)